tests/integration/hxl/metadata.feature

```diff
@@ -0,0 +1,88 @@
+@hxl
+Feature: Testing the HXL Metadata API
+  Scenario: Create a metadata object for an existing export job
+    Given that I want to make a new "HXL Metadata"
+    And that the oauth token is "testadminuser"
+    And that the request "data" is:
+          """
+          {
+              "license_id": 1,
+              "organisation": "ushahidi",
+              "dataset_title": "ushahidi-dataset",
+              "source": "other",
+              "maintainer": "ushahidi-maintainer",
+              "private": true,
+              "export_job_id":1,
+              "user_id":2
```

Comment:
Do we actually need to pass `user_id`? Is it always the current user? If so, we should be pulling it from user context.

Reply:
For reference you can just do this to get it from the session instead: https://github.com/ushahidi/platform/blob/develop/src/Core/Usecase/Contact/CreateContact.php#L19

Reply:
eh , I guess it should always be the current user yea. Changing

Reply:
done